src/VkBootstrap.h

```diff
@@ -115,6 +115,34 @@ template <typename T> class Result {
 	bool m_init;
 };
 
+struct GenericFeaturesPNextNode {
+
+    GenericFeaturesPNextNode() : sType(static_cast<VkStructureType>(0)),
+                                 pNext(nullptr) {
+        memset(fields, 0, sizeof(fields));
+    }
+
+    VkStructureType sType = static_cast<VkStructureType>(0);
+    void* pNext = nullptr;
+    VkBool32 fields[256];
+
+    template <typename T>
+    void set(T const& features) {
+        GenericFeaturesPNextNode node;
```

Comment:
Possible unused variable? I'm not getting a warning though

Reply:
Yep. Unused and removed.